build.sh

```diff
@@ -237,6 +237,13 @@ generate_event_logging_sources()
                     exit
                 fi
                 ;;
+            FreeBSD)
```

Comment:
Can you please get rid of the code duplication by using single case for both Linux and FreeBSD - `Linux|FreeBSD)`. The same in a similar case below.